### PR TITLE
Hide device_id parameter from user in Terraform provider

### PR DESCRIPTION
**Pull request by ziemowit141**

### CODE REVIEW

**Comment by Fenthick** [comment]
Please provider Jira ticket reference

File: synthetics/test_conversions.go [hunk 1]
```diff
@@ -221,7 +220,7 @@ func resourceDataToTest(d *schema.ResourceData) (*synthetics.V202101beta1Test, e
 	test := synthetics.NewV202101beta1Test()
 	test.SetName(d.Get("name").(string))
 	test.SetType(d.Get("type").(string))
-	test.SetDeviceId(d.Get("device_id").(string))
+	test.SetDeviceId("0")
```
Comment (Fenthick) [inline synthetics/test_conversions.go:223]: Please put a comment why Device ID is hardcoded to 0 and not taken from the user

**Review by mmac-m3a — DISMISSED** [review]
LGTM. Please address Daniel's comment.

File: synthetics/test_conversions.go (hunk 1 above)
Comment (ziemowit141) [reply synthetics/test_conversions.go]: Addressed: c564f604a57155de603151b7e17f528bfdcb00d8

**Review by Fenthick — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**